### PR TITLE
Feature/79878 add action to tickets email template

### PR DESCRIPTION
**Pull request by niconerd**
https://central.tri.be/issues/79878

### CODE REVIEW

File: src/views/tickets/email.php [hunk 1]
```diff
@@ -314,6 +314,7 @@
 				<table class="content" align="center" width="620" cellspacing="0" cellpadding="0" border="0" bgcolor="#ffffff" style="margin:0 auto; padding:0;<?php echo $break; ?>">
 					<tr>
 						<td align="center" valign="top" class="wrapper" width="620">
+							<?php do_action( 'tribe_tickets_ticket_email_ticket_top', $ticket ); ?>
```
Comment (bordoni) [inline src/views/tickets/email.php]: We want to add a docblock to this action.
E.g.: https://github.com/moderntribe/event-tickets/blob/master/src/Tribe/Tickets_Handler.php#L1123-L1132

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Whitespace found at end of line | [email.php:317](https://github.com/moderntribe/event-tickets/blob/f1e74ee2ba13b84d11679a83692674b6a891bb99/src/views/tickets/email.php#L317)
:x: | Whitespace found at end of line | [email.php:323](https://github.com/moderntribe/event-tickets/blob/f1e74ee2ba13b84d11679a83692674b6a891bb99/src/views/tickets/email.php#L323)

> via jenkins/codesniffer-phpcs

File: src/views/tickets/email.php (hunk 1 above)
Comment (niconerd) [reply src/views/tickets/email.php]: @bordoni thanks G. Docblock added

**Comment by niconerd** [comment]
Sorry, @bordoni looks like it needs a new review after fixing the changelog merge conflicts

**Pull request merged**